src/appkit.cs

```diff
@@ -4098,6 +4098,16 @@ partial interface NSColor : NSCoding, NSCopying, NSSecureCoding, NSPasteboardRea
 		[Export ("systemGrayColor", ArgumentSemantic.Strong)]
 		NSColor SystemGrayColor { get; }
 
+		[Mac (10, 15)]
+		[Static]
+		[Export ("systemIndigoColor", ArgumentSemantic.Strong)]
+		NSColor SystemIndigoColor { get; }
+
+		[Mac (10, 12)]
```

Comment:
hmm... I'm always wary of new Xcode adding old stuff
make sure you follow up, after the merge, if the old-macos bots are happy

Reply:
Or play it safe, make it 10.15 cause that's when it appeared for the first time and if it's really needed on lower platforms we can re-adjust later.